model/schema/expression.yaml

```diff
@@ -511,6 +513,20 @@ slots:
     range: GOTerm
     multivalued: false
 
+  cellular_component_ribbon_term:
+    description: >-
+      The primary cellular component slim term referred to within the context of an AnatomicalSite object and used by the expression ribbon.
+    domain: AnatomicalSite
+    range: GOTerm
+    multivalued: false
+
+  cellular_component_other:
+    description: >-
+      Is set to true when no cellular_component_ribbon_term is available
+    domain: AnatomicalSite
+    range: boolean
```

Comment:
Is this range correct?

Reply:
It matches other boolean values in the model like [here](https://github.dev/alliance-genome/agr_curation_schema/blob/b25a1dfb8f019d69d571664ec4b6e10dd877b8f9/model/schema/allele.yaml#L1024) so I think so.